CHANGELOG.md

```diff
@@ -10,12 +10,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the TrustConst post-processor from the codebase due to non-usage. This
   change aims to streamline the code and eliminate unnecessary components.
 
-### Fixed 
+### Fixed
 - Fixed issue where `s` was being updated incorrectly on each iteration  of `DLSolver._solve()`.
+- Fixed the calculation of `solve_time` and `pp_time` for all solvers to reflect the time
+for a single calculation of an instance intead of the time to solve the full batch
 
 ### Added
 - Added `PumpedLangevinSolver`, which is an extension of `LangevinSolver` to simulate pumped Langevin dynamics with a demo script in the examples directory.
 - Implemented a simple gradient descent post-processing step, as described in the paper, similar to Langevin dynamics but without noise; uses the Euler method with box constraint imposition at each iteration.
+- Added a saturation parameter `S` as the input to the `langevin_solver`
```

Comment:
It looks like S was also removed from the `__init__` function, which makes this a breaking change. Can you please include that information and move this point up to the breaking changes section?

Reply:
Moreover, I think we should separate this change into a separate PR, so that when it is merged the commit logs will more clearly show which commit introduced this breaking change just by the name.

Reply:
Sounds good! I will create a new PR and issue for this change and log in the breaking changes section.